mslib/utils.py

```diff
@@ -101,6 +101,19 @@
 UR.define("pptv = 1e-12 fraction")
 
 
+def windows_subprocess_startupinfo():
```

Comment:
you could simplify the code quite a bit by returning "None" for non-"nt" OS and always supplying it as an argument.
Alternatively, we could shim the subprocess.run command (https://en.wikipedia.org/wiki/Shim_(computing))

Functionally this is fine.

Reply:
I first thought I just replace it by EasyProcess, https://github.com/ponty/EasyProcess
But that is an API change and it won't go into 5.0.1. I packaged this for conda-forge for pyvirtualdisplay as a dependency. 

That this happens now only on windows desktop start is the reason I choosed that filter. IBut anyway if it breaks after next installation something else we get faster a 5.0.2.